paddle/fluid/framework/ir/graph_pattern_detector.cc

```diff
@@ -1306,6 +1306,55 @@ PDNode *patterns::ConvAffineChannel::operator()(
   return ac_out_var;
 }
 
+PDNode *patterns::TransposeFlattenConcat::operator()(PDNode *conv_in[],
+                                                     int times) {
+  // {trans, trans_out, flatten, flatten_out} * 4
+  PDNode *nodes[times * 4];
```

Comment:
where is `4` comes from, no magic digit here.
Use a const variable instead, and add some comments.